tests/integration/interface_common_test.py

```diff
@@ -34,3 +39,32 @@ def test_iface_description_removal(eth1_up):
     libnmstate.apply(desired_state)
     current_state = statelib.show_only(('eth1',))
     assert Interface.DESCRIPTION not in current_state[Interface.KEY][0]
+
+
+def test_take_over_virtual_interface_then_remove():
+    dummy_name = 'dummy_test'
+    assert (
+        libcmd.exec_cmd(['ip', 'link', 'add', dummy_name, 'type', 'dummy'])[0]
+        == 0
+    )
+    dummy_desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: dummy_name,
+                Interface.STATE: InterfaceState.UP,
+                Interface.IPV4: {InterfaceIPv4.ENABLED: False},
+                Interface.IPV6: {InterfaceIPv6.ENABLED: False},
+            }
+        ]
+    }
+    libnmstate.apply(dummy_desired_state)
+    
```

Comment:
There seems to be trailing whitespace in this line according to my editor.

Reply:
Done, thanks